study-builder/studies/mpc/study-events.conf

```diff
@@ -0,0 +1,434 @@
+              "type": "ANSWER",
+              "questionStableId": "PREQUAL_LAST_NAME"
+            },
+            "target": {
+              "type": "PARTICIPANT_PROFILE_LAST_NAME"
+            }
+          }
+        ]
+      },
+      "dispatchToHousekeeping": false,
+      "order": 1
+    },
+
+    # activity instance creation events
+    {
+      "trigger": {
+        "type": "ACTIVITY_STATUS",
+        "activityCode": "PREQUAL",
+        "statusType": "COMPLETE"
+      },
+      "action": {
+        "type": "ACTIVITY_INSTANCE_CREATION",
+        "activityCode": "ABOUTYOU"
+      },
+      "cancelExpr": """
+        !user.studies["cmi-mpc"].forms["PREQUAL"].questions["PREQUAL_SELF_DESCRIBE"].answers.hasOption("DIAGNOSED")
+      """,
+      "maxOccurrencesPerUser": 1,
+      "dispatchToHousekeeping": false,
+      "order": 1
+    },
+    {
+      "trigger": {
+        "type": "ACTIVITY_STATUS",
+        "activityCode": "ABOUTYOU",
+        "statusType": "COMPLETE"
+      },
+      "action": {
+        "type": "ACTIVITY_INSTANCE_CREATION",
+        "activityCode": "CONSENT"
+      },
+      "cancelExpr": """
+        !user.studies["cmi-mpc"].forms["ABOUTYOU"].questions["COUNTRY"].answers.hasAnyOption("US", "CA", "PR", "GU", "VI", "MP", "AS")
+      """,
+      "maxOccurrencesPerUser": 1,
+      "dispatchToHousekeeping": false,
+      "order": 1
+    },
+    {
+      "trigger": {
+        "type": "ACTIVITY_STATUS",
+        "activityCode": "CONSENT",
+        "statusType": "COMPLETE"
+      },
+      "action": {
+        "type": "ACTIVITY_INSTANCE_CREATION",
+        "activityCode": "RELEASE"
+      },
+      "maxOccurrencesPerUser": 1,
+      "dispatchToHousekeeping": false,
+      "order": 1
+    },
+
+    # registration event
+    {
+      "trigger": {
+        "type": "USER_REGISTERED",
+      },
+      "action": {
+        "type": "SENDGRID_EMAIL",
+        "emailTemplate": ${emails.welcome},
+        "linkedActivityCode": "ABOUTYOU",
+        "language": "en",
+        "pdfAttachments": []
+      },
+      "cancelExpr": """
+        !user.studies["cmi-mpc"].forms["PREQUAL"].hasInstance()
+        || !user.studies["cmi-mpc"].forms["PREQUAL"].questions["PREQUAL_SELF_DESCRIBE"].answers.hasOption("DIAGNOSED")
+      """,
+      "maxOccurrencesPerUser": 1,
+      "dispatchToHousekeeping": true,
+      "order": 1
+    },
+
+    # activity status email events
+    {
+      "trigger": {
+        "type": "ACTIVITY_STATUS",
+        "activityCode": "CONSENT",
+        "statusType": "CREATED"
+      },
+      "action": {
+        "type": "SENDGRID_EMAIL",
+        "emailTemplate": ${emails.aboutyouCompleted},
+        "linkedActivityCode": "CONSENT",
+        "language": "en",
+        "pdfAttachments": []
+      },
+      "maxOccurrencesPerUser": 1,
+      "dispatchToHousekeeping": true,
+      "order": 1
+    },
+    {
+      "trigger": {
+        "type": "ACTIVITY_STATUS",
+        "activityCode": "CONSENT",
+        "statusType": "COMPLETE"
```

Comment:
Maybe this should be `RELEASE, CREATED` so we don't run into any timing issues? Either that or set `order: 2` so it comes after Release is created.